app/controllers/users_controller.rb

```diff
@@ -1,4 +1,24 @@
 class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+  end
+
   def new
+    @user = User.new
+  end
+  def create
```

Comment:
```suggestion
  end

  def create
```
please add new lines between methods, ditto for rest of this file

Reply:
Noted added to all controllers and tests

Reply:
added spaces